tools/benchmark/cmd/put.go

```
@@ -90,6 +96,21 @@ func putFunc(cmd *cobra.Command, args []string) {
 		close(requests)
 	}()
 
+	if compactInterval > 0 {
+		go func() {
+			for {
+				time.Sleep(compactInterval)
+				st := time.Now()
+				err := compactKV(clients)
+				var errStr string
+				if err != nil {
+					errStr = err.Error()
+				}
+				results <- result{errStr: errStr, duration: time.Since(st)}
```

Ok will ignore the errors. Haven't got any in my test runs.